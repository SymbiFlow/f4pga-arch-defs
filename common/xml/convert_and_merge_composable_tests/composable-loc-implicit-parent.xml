<?xml version="1.0"?>
<!--
  Convert
    <loc ...><port ...><port ...></loc>
  to
    <loc ...>BLOCK.PORT BLOCK.PORT</loc>
  -->
<xml>
 <pb_type name="parent">
  <input name="i" width="2" />
  <pinlocations>
   <loc>
    <port name="o" />
    <port name="i" bit="0" />
    <port name="i" bit="1" />
   </loc>
  </pinlocations>
  <random>
   <loc>
    <port name="o" />
   </loc>
  </random>
 </pb_type>
</xml>
