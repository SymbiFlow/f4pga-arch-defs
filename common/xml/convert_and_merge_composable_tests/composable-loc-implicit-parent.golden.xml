<?xml version="1.0"?>
<xml>
  <pb_type name="parent">
    <input name="i" width="2"/>
    <pinlocations>
      <loc>
parent.o
parent.i[0]
parent.i[1]
</loc>
    </pinlocations>
    <random>
      <loc>
        <port name="o"/>
      </loc>
    </random>
  </pb_type>
</xml>
